diff --git a/server/o3dgc/src/CMakeLists.txt b/server/o3dgc/src/CMakeLists.txt
index 90ef77a..a657a07 100644
--- a/server/o3dgc/src/CMakeLists.txt
+++ b/server/o3dgc/src/CMakeLists.txt
@@ -5,4 +5,4 @@ set(CMAKE_COMMON_INC "${${PROJECT_NAME}_SOURCE_DIR}/../scripts/cmake_common.cmak
 add_subdirectory ("${${PROJECT_NAME}_SOURCE_DIR}/o3dgc_common_lib")
 add_subdirectory ("${${PROJECT_NAME}_SOURCE_DIR}/o3dgc_encode_lib")
 add_subdirectory ("${${PROJECT_NAME}_SOURCE_DIR}/o3dgc_decode_lib")
-add_subdirectory ("${${PROJECT_NAME}_SOURCE_DIR}/test")
\ No newline at end of file
+# add_subdirectory ("${${PROJECT_NAME}_SOURCE_DIR}/test")
\ No newline at end of file
